package.json

```diff
@@ -11,13 +11,11 @@
     },
     ".": {
       "import": "./lib/mjs/index.js",
-      "require": "./lib/cjs/index.js",
-      "types": "./lib/types/index.d.ts"
+      "require": "./lib/cjs/index.js"
```

Comment:
The types import was ignored here. When I updated esbuild it threw the warning.

We already export all the types globally [here](https://github.com/Constructor-io/constructorio-ui-autocomplete/pull/207/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L28)